src/gax.ts

```diff
@@ -262,7 +307,11 @@ export class CallSettings {
       isBundling = options.isBundling!;
     }
 
-    if ('maxRetries' in options) {
+    if ('maxRetries' in options && typeof options.maxRetries !== 'undefined') {
```

Comment:
In modern JS / TS it's normally safe to say `if (options.maxRetries !== undefined)` without using `typeof`. I know this code base uses `typeof` a lot, but moving forward, we can write simpler JS code.

Or maybe just `if (options.maxRetries)`, should also work!

Reply:
I'll be sure to steer away from `typeof`! I think I want to leave it as `!==` just in case someone passes a zero for `maxRetries`.